src/plugins/index.ts

```diff
@@ -0,0 +1,36 @@
+import * as FastifySwagger from 'fastify-swagger' 
+import * as Fastify from 'fastify'
```

Comment:
I suggest you change this line to:
```suggestion
/* eslint-disable-next-line no-unused-vars */
import { FastifyInstance } from 'fastify'
```